apps/btadv/app.js

```diff
@@ -379,12 +379,12 @@ var setIntervals = function (locked, connected) {
     if (connected === void 0) { connected = NRF.getSecurityStatus().connected; }
     changeInterval(redrawInterval, locked ? 15000 : 5000);
     if (connected) {
-        var interval = btnsShown ? 5000 : 1000;
+        var interval_1 = btnsShown ? 5000 : 1000;
```

Comment:
Was there a reason for this change? It seems like it might be accidental? I'd be pretty sure it won't have any effect anyway

Reply:
Ah yes - I've backed this out, `tsc` seems to add it in if it finds a conflicting variable, although it doesn't seem too smart about it.